geaflow/geaflow-dsl/geaflow-dsl-plan/src/test/java/com/antgroup/geaflow/dsl/udf/string/UDFStringTest.java

```diff
@@ -69,10 +70,48 @@ public void testBase64() {
     @Test
     public void testConcat() {
         String string = "ant group";
+        BinaryString binaryString = BinaryString.fromString(string);
         Concat test = new Concat();
         test.open(null);
-        assertEquals(test.eval(string, string, string),
+        assertEquals(test.eval(string, string, string), "ant groupant groupant group");
+        assertEquals(test.eval(binaryString, binaryString, binaryString),
             "ant groupant groupant group");
+        assertEquals(test.eval(BinaryString.fromString("蚂蚁1"), BinaryString.fromString("蚂蚁2"),
+            BinaryString.fromString("蚂蚁3")), "蚂蚁1蚂蚁2蚂蚁3");
+    }
+
+    @Test
+    public void testConcatWS() {
+        String string = "ant group";
+        ConcatWS test = new ConcatWS();
+        BinaryString binaryString = BinaryString.fromString(string);
+        test.open(null);
+        assertEquals(test.eval(",", string, string, string),
+            "ant group,ant group,ant group");
+        assertEquals(test.eval("-", string, string, string),
+            "ant group-ant group-ant group");
+        assertEquals(test.eval("***", string, string, string),
+            "ant group***ant group***ant group");
+        assertEquals(test.eval("\t", string, string, string),
+            "ant group\tant group\tant group");
+        assertEquals(test.eval("\n", string, string, string),
+            "ant group\nant group\nant group");
+
+        assertEquals(
```

Comment:
It's necessary to add a method that takes a separator as a normal String type？

Reply:
removed~